lib/FrameManager.js

```diff
@@ -259,17 +254,19 @@ FrameManager.Events = {
  */
 class Frame {
   /**
+   * @param {!FrameManager} frameManager
    * @param {!Puppeteer.CDPSession} client
    * @param {?Frame} parentFrame
    * @param {string} frameId
    */
-  constructor(client, parentFrame, frameId) {
+  constructor(frameManager, client, parentFrame, frameId) {
+    this._frameManager = frameManager;
```

Comment:
page

Reply:
done

Reply:
Not done; we need framemanager in element handle to resolve content frames